app/oxalis/telemetry/SlackNotificationService.scala

```diff
@@ -48,4 +48,10 @@ class SlackNotificationService @Inject()(rpc: RPC, config: WkConf) extends LazyL
       title = "Task creation with base",
       msg = s"$numberOfTasks tasks with BaseAnnotation for TaskTypes ${taskType.mkString(", ")} have been created"
     )
+
+  def noticeSlowRequest(msg: String): Unit =
```

Comment:
This was already used before this PR, how is this possible? :thinking:

Reply:
There is also a DSSlackNotificationService and a TSSlackNotificationService. This could be made DRYer by using a trait, but since the classes are small, I think it’s fine for the moment.
The wk-side one didn’t have this function before.